src/resources/payment_log_report.js

```diff
@@ -0,0 +1,8 @@
+import reportBase from './report_base';
+
+export default api => (
+  class PaymentLogReport extends reportBase(api) {
+    static _name = 'PaymentLogReport';
+    static url = 'reports/payment_log';
```

Comment:
It could, if you refactored a little like this:


```javascript
export default api => {
  const Base = reportBase(api);

  return class PaymentLogReport extends Base {
    static _name = 'PaymentLogReport';
    static url = `${Base.url}/payment_log`;
    // ...
  }
}
```